packages/core-components/src/components/ProgressBars/GaugeCard.tsx

```diff
@@ -75,15 +81,22 @@ export function GaugeCard(props: Props) {
   };
 
   return (
-    <Box className={classes.root}>
+    <Box className={size === 'small' ? classes.rootSmall : classes.root}>
       <InfoCard
         title={title}
         subheader={subheader}
         deepLink={deepLink}
         variant={variant}
         icon={icon}
+        titleTypographyProps={
+          size === 'small'
+            ? {
+                variant: 'h6',
+              }
+            : undefined
+        }
```

Comment:
related to the `25px` question above: curious why the title `variant` is `h6` here & how that relates to the `25px`

Reply:
So 25px and 45px was chosen manually based on what fits inside the circle of the `<Gauge>` (at least I think that's how 45px was chosen originally). `h6` here reflects the title of the MUI `<CardHeader>`, which is shrunken from the default, to better fit a smaller version of the card.
Both of them are somewhat arbitrary, but having a fixed px for the `<Gauge>` can make sense, so that other font overriding things don't mess with it, making it overflow or shrink, while the title font doesn't really have the same concern.

Reply:
ok. should be fine. heads-up that the default size for the `h6` variant is `20px` - so it will be smaller than the percentage: https://m2.material.io/design/typography/the-type-system.html#type-scale

Reply:
Thanks for the info! I think that's actually good with smaller text, so you can more easily fit longer texts without too many line breaks. These gauges can represent e.g. tech insights aggregations with texts like "Dependabot critical level" and even longer texts.